README.md

```diff
@@ -64,9 +40,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- MIcroverse for the UI template
 
 ## 📝 License
 
```

Comment:
### Your MIT.md
![empty MIT file](https://user-images.githubusercontent.com/93039359/182381542-d6b50f2c-22fb-442b-b94c-18faba355063.png)

### Example MIT.md
![sample](https://user-images.githubusercontent.com/93039359/182381778-ff96c7c8-9b06-42ee-8146-7fa3dcafe6f3.png)


- [ ] It's great that you have added an MIT link here and you have added a MIT.md file in your project but your MIT.md file is empty means it doesn't contain your name, current year, and APP-TYPE. Please add these things :+1: